src/quacc/recipes/espresso/core.py

```diff
@@ -350,3 +350,60 @@ def non_scf_job(
         additional_fields={"name": "pw.x Non SCF"},
         copy_files=prev_dir,
     )
+
+
+@job
+def bands_job(
+    atoms: Atoms,
+    prev_dir: str | Path,
+    preset: str | None = "sssp_1.3.0_pbe_efficiency",
+    parallel_info: dict[str] | None = None,
+    test_run: bool = False,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic bands calculation with pw.x.
+
+    Parameters
+    ----------
+    atoms
+        The Atoms object.
+    prev_dir
+        Outdir of the previously ran pw.x calculation. This is used to copy
+        the entire tree structure of that directory to the working directory
+        of this calculation.
```

Comment:
Could be helpful to mention what needs to be present in the previous directory. It's not just any `pw.x` calculation --- presumably it must be one with the charge density, right?